person.rb

```diff
@@ -0,0 +1,24 @@
+# rubocop:disable Naming/PredicateName
```

Comment:
- [ ] Kindly remove this line & solve the linter manually.

Reply:
@sja-thedude Kindly, do you advice I change the name of the function since that seems to be the only way to fix the linter issues.